phpcs.xml.dist

```diff
@@ -1,8 +1,307 @@
+        <exclude-pattern>src/Consumer.php</exclude-pattern>
+        <exclude-pattern>src/Token/Access.php</exclude-pattern>
+        <exclude-pattern>src/Token/AbstractToken.php</exclude-pattern>
+    </rule>
+    <rule ref="WebimpressCodingStandard.PHP.CorrectClassNameCase.InvalidInPhpDocs">
+        <exclude-pattern>src/Config/StandardConfig.php</exclude-pattern>
+        <exclude-pattern>src/Token/AbstractToken.php</exclude-pattern>
+    </rule>
+    <rule ref="Generic.Arrays.ArrayIndent.CloseBraceIncorrect">
+        <exclude-pattern>src/Config/StandardConfig.php</exclude-pattern>
+    </rule>
+    <rule ref="SlevomatCodingStandard.Variables.UselessVariable.UselessVariable">
+        <exclude-pattern>src/Http/RequestToken.php</exclude-pattern>
+        <exclude-pattern>src/Http/AccessToken.php</exclude-pattern>
+        <exclude-pattern>src/Signature/Plaintext.php</exclude-pattern>
+    </rule>
+    <rule ref="SlevomatCodingStandard.Operators.DisallowEqualOperators.DisallowedNotEqualOperator">
+        <exclude-pattern>src/Http/Utility.php</exclude-pattern>
+    </rule>
+    <rule ref="PSR12.ControlStructures.ControlStructureSpacing.FirstExpressionLine">
+        <exclude-pattern>src/Http.php</exclude-pattern>
+        <exclude-pattern>src/Config/StandardConfig.php</exclude-pattern>
+        <exclude-pattern>src/Signature/AbstractSignature.php</exclude-pattern>
+        <exclude-pattern>src/Token/Access.php</exclude-pattern>
+        <exclude-pattern>src/Token/AbstractToken.php</exclude-pattern>
+    </rule>
+    <rule ref="Squiz.Commenting.FunctionComment.MissingParamName">
+        <exclude-pattern>src/Http.php</exclude-pattern>
+        <exclude-pattern>src/Consumer.php</exclude-pattern>
+    </rule>
+    <rule ref="PSR12.Properties.ConstantVisibility.NotFound">
+        <exclude-pattern>src/OAuth.php</exclude-pattern>
+        <exclude-pattern>src/Token/AbstractToken.php</exclude-pattern>
+    </rule>
+    <rule ref="SlevomatCodingStandard.Commenting.RequireOneLinePropertyDocComment.MultiLinePropertyComment">
+        <exclude-pattern>src/Consumer.php</exclude-pattern>
+        <exclude-pattern>src/Token/AuthorizedRequest.php</exclude-pattern>
+        <exclude-pattern>src/Token/AbstractToken.php</exclude-pattern>
+    </rule>
+    <rule ref="SlevomatCodingStandard.Commenting.DocCommentSpacing.IncorrectLinesCountBetweenDescriptionAndAnnotations">
+        <exclude-pattern>src/Signature/AbstractSignature.php</exclude-pattern>
+    </rule>
+    <rule ref="PSR12.ControlStructures.ControlStructureSpacing.FirstExpressionLine">
+        <exclude-pattern>src/Token/AuthorizedRequest.php</exclude-pattern>
+    </rule>
+    <rule ref="SlevomatCodingStandard.Namespaces.UnusedUses.UnusedUse">
+        <exclude-pattern>src/Token/TokenInterface.php</exclude-pattern>
+    </rule>
+    <rule ref="SlevomatCodingStandard.TypeHints.ReturnTypeHint.MissingAnyTypeHint">
+        <exclude-pattern>src/Token/AbstractToken.php</exclude-pattern>
+    </rule>
```

Comment:
Similar question to https://github.com/laminas/laminas-file/pull/15#pullrequestreview-1729323169

/cc @laminas/technical-steering-committee

Reply:
@Ocramius @gsteel 
I have addressed in this PR similar queries  https://github.com/laminas/laminas-file/pull/15#pullrequestreview-1729323169, Now can you please guide now how to proceed on this?